cmd/join_test.go

```diff
@@ -0,0 +1,74 @@
+package cmd
+
+import (
+	"testing"
+)
+
+type test struct {
+	serverIP       string
+	joinToken      string
+	installStr     string
+	k3sExtraArgs   string
+	serverAgent    bool
+	installk3sExec string
+}
+
+func Test_makeJoinServerExec(t *testing.T) {
+	tests := []test{
+		{
+			serverIP:       "172.27.251.164",
```

Comment:
For the test case I'd like to see one more thing

`testCase` or `scenario` with a description.

This increases readability, and means no brain power is needed to understand the scenario.